lightning-invoice/src/payment.rs

```diff
@@ -108,14 +116,14 @@
 //!     match event {
 //!         Event::PaymentPathFailed { .. } => println!("payment failed after retries"),
 //!         Event::PaymentSent { .. } => println!("payment successful"),
-//!         _ => {},
+//!         _ => {}
```

Comment:
Leave comma for consistency with the other arms.

Reply:
Addressed in c5206b3bf646b61db957ef7dd80b79538114181d